### PR TITLE
test(): updated protractor test enviroment and added registration test

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -117,6 +117,7 @@
     "enhanced-resolve": "^3.4.1",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",
+    "has-flag": "^2.0.0",
```
Comment (jkuri) [inline package.json]: please don't add any additional libraries. if there's a need please consult with the whole team.

File: tests/e2e/tests/010-GitHub/340-PingEvent/010.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { request, header } from '../../../webhooks/github/PingEvent';
+import { sendRequest } from '../../../utils/utils';
+
+export default function() {
+  return Promise.resolve()
+    .then(() => sendRequest(request, header))
+    .then(parsedBody => {
+      // console.log(parsedBody);
```
Comment (jkuri) [inline tests/e2e/tests/010-GitHub/340-PingEvent/010.ts]: remove comments please.

File: tests/protractor_runner.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { execSilent, exec, abstruse, killAllProcesses } from './helpers/process';
+import { execSilent, exec, abstruse, killAllProcesses } from './e2e//utils/process';
```
Comment (jkuri) [inline tests/protractor_runner.ts]: typo: `//`

**Comment by Izak88** [comment]
updated after code review

**Pull request merged**